app/models/manageiq/providers/openstack/cloud_manager.rb

```diff
@@ -181,4 +181,19 @@ def vm_delete_evm_snapshot(vm, image_id)
   def self.event_monitor_class
     ManageIQ::Providers::Openstack::CloudManager::EventCatcher
   end
+
+  #
+  # Statistics
+  #
+
+  # gigabytes
+  def cinder_disk_usage
```

Comment:
I recommend having everything return in bytes.  Keeping a consistent base type makes the UI / reporting a lot easier.